SEED-platform/seed
# Migration Performance

#### Any background context you want to provide?
Migration 0109 can take 4ish hours to recompute all property/taxlot hashes
#### What's this PR do?
This wraps the expensive operations in atomic transactions to save only 2 transactions instead of millions, and cuts the time by at least half
#### How should this be manually tested?
Run the migrations against a database that hasn't already been migrated to this point

**Note that this will require local developers to start over with their database since we changed the order of the migrations.**
